tests/framework/integration/cluster.go

```diff
@@ -553,10 +556,11 @@ type Member struct {
 	Server        *etcdserver.EtcdServer
 	ServerClosers []func()
 
-	GrpcServerOpts []grpc.ServerOption
-	GrpcServer     *grpc.Server
-	GrpcURL        string
-	GrpcBridge     *bridge
+	GrpcServerOpts           []grpc.ServerOption
+	GrpcAdditionalServerOpts []grpc.ServerOption
```

Comment:
Do we really need it here ? 

I think that `MustNewMember` could just propagate `MemberConfig.GrpcAdditionalServerOpts` into `GrpcServerOpts` and not store it here explicitly.